topdown/http_test.go

```diff
@@ -220,3 +220,69 @@ func TestInvalidKeyError(t *testing.T) {
 		runTopDownTestCase(t, data, tc.note, tc.rules, tc.expected)
 	}
 }
+
+// TestHTTPRedirectDisable tests redirects are not enabled by default
+func TestHTTPRedirectDisable(t *testing.T) {
+
+	// expected result
+	expectedResult := make(map[string]interface{})
+	expectedResult["status"] = "301 Moved Permanently"
+	expectedResult["status_code"] = http.StatusMovedPermanently
+	expectedResult["body"] = nil
+
+	resultObj, err := ast.InterfaceToValue(expectedResult)
+	if err != nil {
+		panic(err)
+	}
+
+	var testURL = "http://google.com"
+
+	// run the test
+	tests := []struct {
+		note     string
+		rules    []string
+		expected interface{}
+	}{
+		{"http.send", []string{fmt.Sprintf(
+			`p = x { http.send({"method": "get", "url": "%s"}, x) }`, testURL)}, resultObj.String()},
+	}
```

Comment:
[nit] It feels a bit odd to set up the struct slice and then only have one test case. But it surely doesn't hurt.